zenodo/modules/records/serializers/to_marc21/rules.py

```diff
@@ -107,3 +107,18 @@ def reverse_conkerence_url(dummy_self, key, value):
         '$ind1': '4',
         '$ind2': '_',
     }
+
+
+@to_marc21.over('856', '^_files$')
+@utils.reverse_for_each_value
+@utils.filter_values
+def reverse_files(dummy_self, key, value):
+    """Reverse - Files."""
+    return {
+        's': str(value['size']),
+        'u': value['uri'],
+        'w': value['checksum'],
```

Comment:
In the old MARC export we didn't include the checksum and using [**$w - Record control number**](https://www.loc.gov/marc/bibliographic/bd856.html) I don't think would be correct.

Also, we did add **$z - Public note** with the ordering of the item, but I think we can leave it out.